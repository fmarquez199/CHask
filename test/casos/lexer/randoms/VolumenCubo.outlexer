"[(world), pos (1,1),(%VolumenCubo%), pos (1,7),(:), pos (1,20),(\\n), pos (1,21),(\\n), pos (2,38),(Skill), pos (3,3),Identificador \"arista\", pos (3,9),(,), pos (3,15),Identificador \"volumen\", pos (3,17),(\\n), pos (3,24),Identificador \"arista\", pos (5,3),(=), pos (5,10),(joystick), pos (5,12),String \"~Introduzca arista: ~\", pos (5,21),(\\n), pos (5,42),Identificador \"volumen\", pos (7,3),(=), pos (7,11),Identificador \"arista\", pos (7,13),(*), pos (7,20),Identificador \"arista\", pos (7,22),(*), pos (7,29),Identificador \"arista\", pos (7,31),(\\n), pos (7,37),(drop), pos (9,3),String \"~el volumen del cubo es: ~\", pos (9,8),(,), pos (9,34),Identificador \"volumen\", pos (9,36),(\\n), pos (9,43),(.~), pos (10,1),(\\n), pos (10,3)]"
