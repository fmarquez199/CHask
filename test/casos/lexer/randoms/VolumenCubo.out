Token world en la fila: 1, columna: 1
Token nombre programa %VolumenCubo% en la fila: 1, columna: 7
Token : en la fila: 1, columna: 20
Token End Instruction \n en la fila: 1, columna: 21
Token End Instruction \n en la fila: 2, columna: 38
Token Skill en la fila: 3, columna: 3
Token identificador "arista" en la fila: 3, columna: 9
Token , en la fila: 3, columna: 15
Token identificador "volumen" en la fila: 3, columna: 17
Token End Instruction \n en la fila: 3, columna: 24
Token End Instruction \n en la fila: 4, columna: 3
Token identificador "arista" en la fila: 5, columna: 3
Token = en la fila: 5, columna: 10
Token joystick en la fila: 5, columna: 12
Token string ~Introduzca arista: ~ en la fila: 5, columna: 21
Token End Instruction \n en la fila: 5, columna: 42
Token End Instruction \n en la fila: 6, columna: 3
Token identificador "volumen" en la fila: 7, columna: 3
Token = en la fila: 7, columna: 11
Token identificador "arista" en la fila: 7, columna: 13
Token * en la fila: 7, columna: 20
Token identificador "arista" en la fila: 7, columna: 22
Token * en la fila: 7, columna: 29
Token identificador "arista" en la fila: 7, columna: 31
Token End Instruction \n en la fila: 7, columna: 37
Token End Instruction \n en la fila: 8, columna: 3
Token drop en la fila: 9, columna: 3
Token string ~el volumen del cubo es: ~ en la fila: 9, columna: 8
Token , en la fila: 9, columna: 34
Token identificador "volumen" en la fila: 9, columna: 36
Token End Instruction \n en la fila: 9, columna: 43
Token .~ en la fila: 10, columna: 1
Token End Instruction \n en la fila: 10, columna: 3
